web/flask/template/index.html

```diff
@@ -393,6 +397,10 @@
             <label>Message</label>
             <p>Task ID</p>
           </div>
+          <div class="col">
+            <label>Download link</label>
+            <p>Task ID</p>
```

Comment:
Suggest to use `a` tag to make it clickable
```suggestion
            <a href="">Task ID</a>
```